backend/app/Http/Controllers/Web/BackupController.php

```
@@ -0,0 +1,597 @@
+            DB::reconnect();
+
+            // 3. Wipe and re-migrate & seed
+            Artisan::call('migrate:fresh', ['--force' => true]);
+            Artisan::call('db:seed', ['--force' => true]);
+
+            // 4. Reconnect again after migrations to ensure fresh schema states
+            DB::purge();
+            DB::reconnect();
+
+            // 5. Recreate admin user
+            $admin = User::create([
```

PHPStan (2.2.2)

[error] Call to an undefined static method App\Models\User::create().


(staticMethod.notFound)